lib/veri.ml

```diff
@@ -277,6 +264,7 @@ class ['a] t arch dis =
       | Error er ->
         SM.update (fun c -> c#notify_error (`Lifter_error (name, er)))
       | Ok bil ->
+        let bil = Stmt.normalize bil in
```

Comment:
This will not normalize memory operations, as `normalize_exp` defaults to `false`. It is not expressed clearly in the documentation, I will update the docs.